compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToUKernels.cpp

```diff
@@ -80,6 +80,38 @@ static FnNameAndDefAttrs getFnNameAndDefAttrs(RewriterBase &rewriter,
 /// ================== SAME UTILITIES AS IREE LLVMCPU ====================
 /// ======================================================================
 
+/// Utility to fetch the element type as string.
+static std::string typeToString(Type elemType) {
```

Comment:
Isn't there some existing utility function for this?

Reply:
I tried `getMnemonic()` but it didn't work.

I've added a better utility that does the same and is generic enough - let me know if that works.

Reply:
Looks good, thanks